### DISCUSSION TITLE
Some core problems

### DISCUSSION THREAD

**Discussion by pabimuri** [FakeFishGames/Barotrauma#9628] [Code questions and modding support]
First of all, I see how game is keep growing, but I can't say that everything is great. However I hope you will reach better results once
1) Why did you changed voice buttons? Is it so hard to let old players from Rusted Remnants use classic controls?
2) Next request is give back  hosts their possibilities to ban players that disconnected from the server already. You know what I mean, many players love griefering
3) And last REALLY BIG PROBLEM your code is weak. I don't want to curse you, but when players without any rights on server can check journals or inventories of other players - this bad. I don't know how they doing this, but you MUST do something with that. 
Journals is not all of this. I saw how players LOAD files in game crashing it or just drop FPS from 60-70 to 1-10. Can you imagine what they can do if they want? Repeat of FromSoftware I hope you understand

**Comment by PraporS** [comment]
The man speaks facts.

**Comment by mygamingaccount** [comment]
Biggest problem:

You started a discussion thread with an overly generic title
You don't expect me to read all this, do you?
